app/src/main/java/org/mifos/mobile/ui/fragments/ReviewLoanApplicationFragment.kt

```diff
@@ -132,22 +140,26 @@ class ReviewLoanApplicationFragment : BaseFragment() {
         activity?.supportFragmentManager?.popBackStack()
     }
 
-    fun showError(message: String?) = if (!Network.isConnected(activity)) {
-        iv_status.setImageResource(R.drawable.ic_error_black_24dp)
-        tv_status.text = getString(R.string.internet_not_connected)
-        ll_add_loan.visibility = View.GONE
-        ll_error.visibility = View.VISIBLE
-    } else {
-        Toaster.show(rootView, message)
+    fun showError(message: String?): Unit = with(binding) {
+        val isConnected = Network.isConnected(activity)
```

Comment:
I am not sure if you need the `isConnected` variable here

Reply:
okay, will inline it in if condition